pyproject.toml

```diff
@@ -7,10 +7,11 @@ dependencies = [
     "craft-cli>=2.6.0",
     "craft-grammar@git+https://github.com/canonical/craft-grammar@feature/pydantic-2",
     "craft-parts@git+https://github.com/canonical/craft-parts@feature/2.0",
-    "craft-providers@git+https://github.com/canonical/craft-providers@feature/2.0-git-modules",
+    "craft-providers@git+https://github.com/canonical/craft-providers@feature/2.0",
     "snap-helpers>=0.4.2",
     "platformdirs>=3.10",
     "pydantic~=2.0",
+    "license-expression>=30.0.0",
```

Comment:
Do we know/trust this provider?

Reply:
It's owned by `nexB` company from `California` and it's available with Apache2 license.
The source is available under:
https://github.com/nexB/license-expression/